cfe time var name changed in .h but not in code

cfe master 6.6 release
osal 4.2.1a
psp 1.3.0.0


building for VxWorks 6.9 SPARC ut700

kspace/plss_cws/../CFS/cfe/fsw/cfe-core/src/time/cfe_time_tone.c
/home/sduran/AEMU_Workspace/plss_cws/../CFS/cfe/fsw/cfe-core/src/time/cfe_time_tone.c: In function 'CFE_TIME_ToneSend':
/home/sduran/AEMU_Workspace/plss_cws/../CFS/cfe/fsw/cfe-core/src/time/cfe_time_tone.c:132:42: error: 'CFE_TIME_ToneDataCmd_Payload_t' has no member named 'AtToneLeaps'
     CFE_TIME_TaskData.ToneDataCmd.Payload.AtToneLeaps = 

FROM cfe 86ce044, 6.5 release plus a couple of hot fixes, master before 6.6 release, cfe_time_utils.h
typedef struct
{

  CFE_TIME_SysTime_t    AtToneMET;      /* MET at time of tone */
  CFE_TIME_SysTime_t    AtToneSTCF;     /* STCF at time of tone */
--->  int16                 AtToneLeaps;    /* Leap Seconds at time of tone */
  int16                 ClockSetState;  /* Time has been "set" */
  int16                 ClockFlyState;  /* Current fly-wheel state */
  CFE_TIME_SysTime_t    AtToneDelay;    /* Adjustment for slow tone detection */
  CFE_TIME_SysTime_t    AtToneLatch;    /* Local clock latched at time of tone */
  CFE_TIME_SysTime_t    CurrentLatch;   /* Local clock latched just "now" */
  CFE_TIME_SysTime_t    TimeSinceTone;  /* Time elapsed since the tone */
  CFE_TIME_SysTime_t    CurrentMET;     /* MET at this instant */

} CFE_TIME_Reference_t;


FROM cfe master 90fdf9f 6.6 release, cfe_time_utils.h --- var name changed in header, but
not in cfe/fsw/cfe-core/src/time/cfe_time_tone.c:132:42
typedef struct
{

  CFE_TIME_SysTime_t    AtToneMET;      /* MET at time of tone */
  CFE_TIME_SysTime_t    AtToneSTCF;     /* STCF at time of tone */
--->  int16                 AtToneLeapSeconds;    /* Leap Seconds at time of tone */
  int16                 ClockSetState;  /* Time has been "set" */
  int16                 ClockFlyState;  /* Current fly-wheel state */
  CFE_TIME_SysTime_t    AtToneDelay;    /* Adjustment for slow tone detection */
  CFE_TIME_SysTime_t    AtToneLatch;    /* Local clock latched at time of tone */
  CFE_TIME_SysTime_t    CurrentLatch;   /* Local clock latched just "now" */
  CFE_TIME_SysTime_t    TimeSinceTone;  /* Time elapsed since the tone */
  CFE_TIME_SysTime_t    CurrentMET;     /* MET at this instant */

} CFE_TIME_Reference_t;


I do not see the issue with a linux build.  I think it might be associated with
#define CFE_PLATFORM_TIME_CFG_BIGENDIAN
The default is to #undef CFE_PLATFORM_TIME_CFG_BIGENDIAN - makes sense on linux/x86
But for vxworks/SPARC, I would think it should be defined.
"**      If this configuration parameter is defined, the CFE time server will
**      publish time tones with payloads in big-endian order, and time clients
**      will expect the tones to be in big-endian order. "

vxworks/sparc will build if 
#undef CFE_PLATFORM_TIME_CFG_BIGENDIAN

--- a/fsw/cfe-core/src/time/cfe_time_tone.c
+++ b/fsw/cfe-core/src/time/cfe_time_tone.c
@@ -141,8 +141,8 @@ void CFE_TIME_ToneSend(void)
         CFE_MAKE_BIG32(Reference.AtToneSTCF.Seconds);
     CFE_TIME_TaskData.ToneDataCmd.Payload.AtToneSTCF.Subseconds =
         CFE_MAKE_BIG32(Reference.AtToneSTCF.Subseconds);
-    CFE_TIME_TaskData.ToneDataCmd.Payload.AtToneLeaps = 
-        CFE_MAKE_BIG16(Reference.AtToneLeaps);
+    CFE_TIME_TaskData.ToneDataCmd.Payload.AtToneLeapSeconds = 
+        CFE_MAKE_BIG16(Reference.AtToneLeapSeconds);
     CFE_TIME_TaskData.ToneDataCmd.Payload.AtToneState = 
         CFE_MAKE_BIG16(AtToneState);
 
@@ -288,8 +288,8 @@ int32 CFE_TIME_ToneSendMET(CFE_TIME_SysTime_t NewMET)
                 CFE_MAKE_BIG32(Reference.AtToneSTCF.Seconds);
             CFE_TIME_TaskData.ToneDataCmd.Payload.AtToneSTCF.Subseconds =
                 CFE_MAKE_BIG32(Reference.AtToneSTCF.Subseconds);
-            CFE_TIME_TaskData.ToneDataCmd.Payload.AtToneLeaps = 
-                CFE_MAKE_BIG16(Reference.AtToneLeaps);
+            CFE_TIME_TaskData.ToneDataCmd.Payload.AtToneLeapSeconds = 
+                CFE_MAKE_BIG16(Reference.AtToneLeapSeconds);
             CFE_TIME_TaskData.ToneDataCmd.Payload.AtToneState = 
                 CFE_MAKE_BIG16(ClockState);
 
@@ -442,7 +442,7 @@ int32 CFE_TIME_ToneSendGPS(CFE_TIME_SysTime_t NewTime, int16 NewLeaps)
                 CFE_MAKE_BIG32(NewSTCF.Seconds);
             CFE_TIME_TaskData.ToneDataCmd.Payload.AtToneSTCF.Subseconds =
                 CFE_MAKE_BIG32(NewSTCF.Subseconds);
-            CFE_TIME_TaskData.ToneDataCmd.Payload.AtToneLeaps = 
+            CFE_TIME_TaskData.ToneDataCmd.Payload.AtToneLeapSeconds = 
                 CFE_MAKE_BIG16(NewLeaps);
             CFE_TIME_TaskData.ToneDataCmd.Payload.AtToneState = 
                 CFE_MAKE_BIG16(ClockState);
@@ -595,8 +595,8 @@ int32 CFE_TIME_ToneSendTime(CFE_TIME_SysTime_t NewTime)
                 CFE_MAKE_BIG32(NewSTCF.Seconds);
             CFE_TIME_TaskData.ToneDataCmd.Payload.AtToneSTCF.Subseconds =
                 CFE_MAKE_BIG32(NewSTCF.Subseconds);
-            CFE_TIME_TaskData.ToneDataCmd.Payload.AtToneLeaps = 
-                CFE_MAKE_BIG16(Reference.AtToneLeaps);
+            CFE_TIME_TaskData.ToneDataCmd.Payload.AtToneLeapSeconds = 
+                CFE_MAKE_BIG16(Reference.AtToneLeapSeconds);
             CFE_TIME_TaskData.ToneDataCmd.Payload.AtToneState = 
                 CFE_MAKE_BIG16(ClockState);
 
@@ -657,7 +657,7 @@ void CFE_TIME_ToneData(const CFE_TIME_ToneDataCmd_Payload_t *ToneDataCmd)
         CFE_MAKE_BIG32(ToneDataCmd->AtToneSTCF.Seconds);
     CFE_TIME_TaskData.PendingSTCF.Subseconds =
         CFE_MAKE_BIG32(ToneDataCmd->AtToneSTCF.Subseconds);
-    CFE_TIME_TaskData.PendingLeaps = CFE_MAKE_BIG16(ToneDataCmd->AtToneLeaps);
+    CFE_TIME_TaskData.PendingLeaps = CFE_MAKE_BIG16(ToneDataCmd->AtToneLeapSeconds);
     CFE_TIME_TaskData.PendingState = CFE_MAKE_BIG16(ToneDataCmd->AtToneState);
 
     #else /* !CFE_PLATFORM_TIME_CFG_BIGENDIAN */

